hack/aks/Makefile

```diff
@@ -88,21 +87,29 @@ overlay-net-up: ## Create vnet, nodenet subnets
 	$(AZCLI) network vnet create -g $(GROUP) -l $(REGION) --name $(VNET) --address-prefixes 10.0.0.0/8 -o none
 	$(AZCLI) network vnet subnet create -g $(GROUP) --vnet-name $(VNET) --name nodenet --address-prefix 10.10.0.0/16 -o none
 
-
 ##@ AKS Clusters
 
 byocni-up: swift-byocni-up ## Alias to swift-byocni-up
 cilium-up: swift-cilium-up ## Alias to swift-cilium-up
 up: swift-up ## Alias to swift-up
 
 overlay-byocni-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster
+	$(AZCLI) network public-ip create --name "overlay-cni-public-ip" \
```

Comment:
I would try to have variables for all these ip names (ex. "overlay-cni-public-ip") since they need to match in 2 places and they are getting repeated. Just a typo can cause regression. Ditto for all scenarios covered in this pipeline.

Reply:
Done